controls/os_spec.rb

```diff
@@ -236,3 +238,24 @@
     its(:group) { should match(/^root|syslog$/) }
   end
 end
+
+control 'os-12' do
+  impact 1.0
+  title 'Detect vulnerabilities in the cpu-vulnerability-directory'
+  desc 'Check for known cpu vulnerabilities described here: https://www.kernel.org/doc/html/v5.6/admin-guide/hw-vuln/index.html'
+
+  if file(cpuvulndir).exist?
```

Comment:
I think we should use `only_if` here (see https://github.com/dev-sec/ssh-baseline/blob/ebdf3e04a5e818f06242bb7302d52a112814d562/controls/sshd_spec.rb#L511 for an example)

Reply:
That is another layer that is not needed in this context as we don't need to pre-check a daemon.

Reply:
that is true. in this case the condition would be `file(cpuvulndir).exist?`.
The benefit of `only_if` in this case is that the controls get listet even if the condition does not match, but they are listed as "ignored". When using if, the controls are not listed.